docs/logs/using.asciidoc

```diff
@@ -1,112 +1,86 @@
 
 [float]
 [[logs-highlight]]
 === Highlight a phrase in the logs stream
-To find instances of a particular word or phrase in the logs stream, click the Highlights button in the toolbar and enter your search terms.
-
-[role="screenshot"]
-image::logs/images/logs-stream-highlight-box.png[Logs highlight dialog box]
-
-This will highlight any instances of your search terms that appear in the logs stream.
-
-[role="screenshot"]
-image::logs/images/logs-stream-highlight-entries.png[Logs stream with the search term 'safepoint' highlighted]
+To highlight a word or phrase in the logs stream, click *Highlights* and enter your search phrase.
+// ++ Is search case sensitive?
+// ++ Can you search for multiple phrases together, if so, what's the separator?
+// ++ What about special characters? For example, I notice that when searching for "Mozilla/4.0" which appears as written in my logs, "Mozilla" is highlighted, as is "4.0" but "/" isn't. The string "-" (which appears in the logs as written, quotes and all, isn't found at all. Any significance?
 
 [float]
 [[logs-event-inspector]]
 === Inspect a log event
-Click on a log event in the stream to inspect it.
+To inspect a log event, hover over it, then click the *View details* icon image:logs/images/logs-view-event.png[View event icon] beside the event.
+This opens the *Log event document details* fly-out that shows the fields associated with the log event.
 
-[role="screenshot"]
-image::logs/images/logs-stream-click-entry.png[Highlighted log gentry in the stream]
-
-All of the fields associated to the log event are displayed.
-
-[role="screenshot"]
-image::logs/images/log-details-flyover.png[Log event document details]
+To quickly filter the logs stream by one of the field values shown here, click the *View event with filter* icon image:logs/images/logs-view-event-with-filter.png[View event icon] beside the field.
+This automatically adds a search filter to the logs stream to filter the entries by this field and value.
 
-By clicking the icon next to one of the listed values, you can filter the logs stream to display only events corresponding to that particular value.
+In the log event details, click *Actions* to see the other actions related to the event.
+Depending on the event and the features you have installed and configured, you may also be able to:
 
-[role="screenshot"]
-image::logs/images/log-details-filter.png[Filtering from the log event details]
-
-[role="screenshot"]
-image::logs/images/logs-stream-filtered-by-value.png[Log console after filtering by a value in the details]
-
-You can use the Actions menu to view any monitor or APM trace that the log event may be associated with.
-
-[role="screenshot"]
-image::logs/images/log-details-actions.png[Log event document details with actions menu open]
+* Select *View monitor status* to <<uptime-overview, view related uptime information>> in the *Uptime* app.
+// ++ Is "monitor" the best choice of word here?
```

Comment:
I know we briefly discussed this one on Zoom. I feel like just using Uptime would probably be clearer. With "Uptime" there's no ambiguous nature about the product you're linking to, and the word "Uptime" suggests "monitoring" anyway.

Reply:
Thanks. I'll raise an issue.